scrapers/ga/events.py

```diff
@@ -20,8 +36,16 @@ def scrape(self, start=None):
         date_slug = start.strftime(date_format)
 
         url = f"https://www.legis.ga.gov/api/meetings?startDate={date_slug}"
-
-        page = self.get(url).json()
+        token = GAEventScraper.get_token()
```

Comment:
this is a bit unusual for Python, typically either a classmethod or just a separate function outside the class is preferable